src/main/java/com/redvinca/assignment/ecom_backend/constantvariables/Constants.java

```diff
@@ -0,0 +1,75 @@
+	public static final String PRODUCT_NOT_FOUND_WARN = "Product not found for productId: {}";
+	public static final String GET_PRODUCT_BY_ID_ENDED = "getProductById method ended for productId: {}";
+	
+	public static final String CONTROLLER_CREATE_PRODUCT_STARTED = "createProduct method started for product: {}";
+    public static final String CONTROLLER_PRODUCT_CREATED_SUCCESSFULLY = "Product created successfully: {}";
+    public static final String CONTROLLER_FAILED_TO_ADD_ITEM = "Failed to add item";
+    public static final String CONTROLLER_GET_ALL_PRODUCTS_STARTED = "getAllProducts method started";
+    public static final String CONTROLLER_RETRIEVED_PRODUCTS = "Retrieved {} products";
+    public static final String CONTROLLER_GET_PRODUCT_BY_ID_STARTED = "getProductById method started for productId: {}";
+    public static final String CONTROLLER_PRODUCT_FOUND = "Product found for productId: {}";
+    public static final String CONTROLLER_PRODUCT_NOT_FOUND = "Product not found for productId: {}";
+
+    public static final String CONTROLLER_ADD_TO_CART_STARTED = "addToCart method started for productId: {}";
+    public static final String CONTROLLER_ADD_TO_CART_SUCCESS = "Product added to cart successfully for productId: {}";
+    public static final String CONTROLLER_ADD_TO_CART_FAILURE = "Failed to add item to cart for productId: {}";
+    public static final String CONTROLLER_GET_ALL_CART_ITEMS_STARTED = "getAllCartItems method started";
+    public static final String CONTROLLER_GET_ALL_CART_ITEMS_SUCCESS = "Retrieved all cart items successfully, count: {}";
+    public static final String CONTROLLER_GET_ALL_CART_ITEMS_FAILURE = "Failed to retrieve cart items";
+    public static final String CONTROLLER_GET_TOTAL_PRICE_STARTED = "getTotalPrice method started";
+    public static final String CONTROLLER_GET_TOTAL_PRICE_SUCCESS = "Total price calculated successfully: {}";
+    public static final String CONTROLLER_GET_TOTAL_PRICE_FAILURE = "Failed to calculate total price";
+    public static final String CONTROLLER_GET_TOTAL_QUANTITY_STARTED = "getTotalQuantity method started";
+    public static final String CONTROLLER_GET_TOTAL_QUANTITY_SUCCESS = "Total quantity calculated successfully: {}";
+    public static final String CONTROLLER_GET_TOTAL_QUANTITY_FAILURE = "Failed to calculate total quantity";
+    public static final String CONTROLLER_UPDATE_QUANTITY_STARTED = "updateQuantity method started for cartItemId: {} with quantityChange: {}";
+    public static final String CONTROLLER_UPDATE_QUANTITY_ENDED = "updateQuantity method ended for cartItemId: {}";
+    public static final String CONTROLLER_DELETE_ITEM_FROM_CART_STARTED = "deleteItemFromCart method started for cartId: {}";
+    public static final String CONTROLLER_DELETE_ITEM_FROM_CART_ENDED = "deleteItemFromCart method ended for cartId: {}";
+
+    // URL constants
+    public static final String PRODUCT_API_URL = "/v3/api-docs/products";
```

Comment:
Constants which are configurable and can change depending upon environment should be moved to application.properties file